client/state/reader/follows/reducer.js

```diff
@@ -11,10 +11,59 @@ import {
 	READER_FOLLOW,
 	READER_UNFOLLOW,
 	READER_FOLLOWS_RECEIVE,
+	READER_SUBSCRIBE_TO_NEW_POST_EMAIL,
+	READER_SUBSCRIBE_TO_NEW_COMMENT_EMAIL,
+	READER_UPDATE_NEW_POST_EMAIL_SUBSCRIPTION,
+	READER_UNSUBSCRIBE_TO_NEW_POST_EMAIL,
+	READER_UNSUBSCRIBE_TO_NEW_COMMENT_EMAIL,
 } from 'state/action-types';
 import { prepareComparableUrl } from './utils';
 import { createReducer } from 'state/utils';
 
+function updatePostSubscription( state, { payload, type } ) {
```

Comment:
I think i figured out now why I kept staring at this reducer file
In general
- reducers split up what to do to state by the action type.  The general paradigm is a unique function per type
- We created 5 different actions types for updating email subs

Then because most of the work ended up being accessing / immutably returning the nested email structure, we consolidated it all into the `updatePostSubscription`.  updatePostSubscription is essentially a full fledged reducer at this point that ignore all the splitting up work its parent is doing.  I think that makes sense, but its exposed what we should have really done all along.

1 action type: `READER_UPDATE_EMAIL_SUBSCRIPTION`.  payload looking like:
```jsx
{
  blogId, deliveryFrequency, sendComments, sendPosts
}
```

We can keep all of the action creators, and have them construct the correct action.